conf/base.config

```diff
@@ -45,7 +45,7 @@ process {
     }
 
     withName: BWAMEM2_INDEX {
-        cpus    = {}        
+        memory  = { check_max( 100.GB  * task.attempt, 'memory' ) }
```

Comment:
Does `BWAMEM2` really need 100GB even for small assemblies? Similarly for `CRAM_FILTER_ALIGN_BWAMEM2_FIXMATE_SORT`.

Reply:
Decreased this to 25GB, your right it wouldn't need 100.
Done